examples/app-pg/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyodbc==4.0.27
```

Comment:
This isn't required because pydbc is already there in the base image we use in the dockerfile.